src/api/api.h

```diff
@@ -29,6 +29,7 @@ void getDBstats(const int *sock);
 void getUnknownQueries(const int *sock);
```

Comment:
`getUnknownQueries`

was added by https://github.com/pi-hole/FTL/pull/174/commits/541d86bc744c7f46a69738c27ad06faa6b7a0fb9.

Does it still serve a purpose? If so we should document it - or did you still remember the function exists ;-)

Reply:
Yes, I do remember it still and have used it not too long ago in debugging a DNSSEC issue (which turned out to be a misconfiguration in the user's upstream `bind` server). That's also exactly the purpose I see for it and don't think there is a general necessity to document it.